react/components/Slider.tsx

```diff
@@ -27,37 +27,45 @@ const Slider: FC<Props> = ({
   fullWidth,
 }) => {
   const handles = useCssHandles(CSS_HANDLES)
-  const { isMobile, device } = useDevice()
-  const { label } = useSliderState()
+  const { isMobile } = useDevice()
+  const { label, slidesPerPage } = useSliderState()
   const containerRef = useRef<HTMLDivElement>(null)
   const controls = `${label
     .toLowerCase()
     .trim()
     .replace(/ /g, '-')}-items`
 
   useAutoplay(infinite, containerRef)
-  useScreenResize(containerRef, device, infinite)
+  useScreenResize(infinite)
   const { onTouchEnd, onTouchStart, onTouchMove } = useTouchHandlers({
-    totalItems,
     infinite,
   })
+  const shouldBeStaticList = slidesPerPage >= totalItems
 
-  const shouldShowArrows = !!(
-    showNavigationArrows === 'always' ||
-    (showNavigationArrows === 'mobileOnly' && isMobile) ||
-    (showNavigationArrows === 'desktopOnly' && !isMobile)
+  const shouldShowArrows = Boolean(
+    (showNavigationArrows === 'always' ||
+      (showNavigationArrows === 'mobileOnly' && isMobile) ||
+      (showNavigationArrows === 'desktopOnly' && !isMobile)) &&
+      !shouldBeStaticList
   )
-  const shouldShowPaginationDots = !!(
-    showPaginationDots === 'always' ||
-    (showPaginationDots === 'mobileOnly' && isMobile) ||
-    (showPaginationDots === 'desktopOnly' && !isMobile)
+  const shouldShowPaginationDots = Boolean(
+    (showPaginationDots === 'always' ||
+      (showPaginationDots === 'mobileOnly' && isMobile) ||
+      (showPaginationDots === 'desktopOnly' && !isMobile)) &&
+      !shouldBeStaticList
   )
 
   return (
     <section
-      onTouchStart={e => (usePagination ? onTouchStart(e) : null)}
-      onTouchEnd={e => (usePagination ? onTouchEnd(e) : null)}
-      onTouchMove={e => (usePagination ? onTouchMove(e) : null)}
+      onTouchStart={e =>
+        usePagination && !shouldBeStaticList ? onTouchStart(e) : null
```

Comment:
I think it would be nice to change the name of `onTouchStart` to `handleTouchStart`. And this `usePagination` variable is a bit confusing I read this and think that is a hook

Reply:
And you could encapsulate all of this code inside of the callback and always pass it